eslint.config.js

```diff
@@ -1,44 +1,41 @@
-import { fixupConfigRules } from '@eslint/compat';
-import pluginJs from '@eslint/js';
-import * as reactQuery from '@tanstack/eslint-plugin-query';
-import pluginImport from 'eslint-plugin-import';
-import pluginReactConfig from 'eslint-plugin-react/configs/recommended.js';
-import * as reactHooks from 'eslint-plugin-react-hooks';
 import globals from 'globals';
-import tseslint from 'typescript-eslint';
+import eslint from '@eslint/js';
+import ts from 'typescript-eslint';
+import reactRefresh from 'eslint-plugin-react-refresh';
```

Comment:
What do you mean? Because without react-refresh plugin we did not have this warning about multiple exports in files 🤔 

So are we sure that this is exactly the same? If yes it also means that we can ged rid of TODOs related with react-refresh plugin 🤔

Reply:
It just shows in the console when app is running. Yes, we can also can remove TODOs
<img width="1642" alt="Screenshot 2024-07-31 at 2 36 11 PM" src="https://github.com/user-attachments/assets/217290ca-2baa-4ae0-9234-43dd0536046f">

Reply:
I've read the dosc and i noticed this:
https://github.com/vitejs/vite-plugin-react-swc?tab=readme-ov-file#consistent-components-exports

So i think we can leave this. To have warnings earlier, during linting instead of runtime